util/smaractconfig.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+Created on June 18 2020
+
+@author: Anders Muskens
+Copyright © 2020 Anders Muskens, Delmic
+
+This is a script that allows you to change the referencing mode on the Smaract 3DOF stage, and run a reference move. 
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from odemis.driver import smaract
+import sys
+import argparse
+import logging
+import time
+
+NUM_CHANNELS = 3
```

Comment:
Don't hard code this value. Just compute the axes based on the axes in arguments.